src/BuildScriptGeneratorCli/Commands/BuildCommandBase.cs

```diff
@@ -25,13 +25,18 @@ internal abstract class BuildCommandBase : CommandBase
             Description = "The version of the programming platform used in the provided source directory.")]
         public string PlatformVersion { get; set; }
 
-        [Option("--package", CommandOptionType.NoValue,
+        [Option(
+            "--package",
+            CommandOptionType.NoValue,
```

Comment:
why this is `.NoValue` not `.SingleValue`

Reply:
because its'a flag .. check the test tests\Oryx.BuildImage.Tests\Node\NodeVsCodeDependencies.cs, line 78

Reply:
but thinking out loud .. i know it's irrelevant for this pr .. but i think this --package should have a test for every platform, we have it for node but should we have  fail tests for other platforms?